Resources/views/layout.html.twig

```diff
@@ -20,7 +20,7 @@ file that was distributed with this source code.
             {% block sonata_page_breadcrumb %}
                 <div class="row">
                     {% if sonata_seo_context is not defined %}
-                        {% set sonata_seo_context = 'homepage' %}
+                        {% set sonata_seo_context = 'page' %}
```

Comment:
i dont think the locale you think is the used locale, the goal isn't to use the default locale in parameters but the already used locale when requests are send ....